source/slang/slang-emit.cpp

```diff
@@ -1292,6 +1293,13 @@ SlangResult emitSPIRVForEntryPointsDirectly(
     auto artifact = ArtifactUtil::createArtifactForCompileTarget(asExternal(codeGenContext->getTargetFormat()));
     artifact->addRepresentationUnknown(ListBlob::moveCreate(spirv));
 
+#if 0
```

Comment:
Leftover debug code?

Reply:
I just wanted to keep that code disabled to print disassembled code right after code generation before spirv-opt is run.